cmds/info.c

```diff
@@ -1133,6 +1133,27 @@ void process_uftrace_info(struct uftrace_data *handle, struct opts *opts,
 		       info->rblock, info->wblock);
 	}
 	process(data, "\n");
+
+	snprintf(buf, sizeof(buf), "%s/note.txt", opts->dirname);
+	if (!access(buf, F_OK)) {
+		long size;
+		FILE *fp = fopen(buf, "r");
+		char *note_buf;
+
+		fseek(fp, 0, SEEK_END);
+		size = ftell(fp);
+		rewind(fp);
+
+		note_buf = xmalloc(size + 1);
+		if (fread(note_buf, 1, size, fp) != (size_t)size)
+			pr_dbg("couldn't read the whole note.txt contents");
+		note_buf[size] = '\0';
+
+		process(data, "# extra note information\n");
+		process(data, "# ======================\n");
+		puts(note_buf);
+		free(note_buf);
```

Comment:
You need `fclose(fp);` too.

Reply:
I thought I added, but not updated here. just added it. Thanks.